pkgs/by-name/ut/ut1999/package.nix

```
@@ -47,34 +53,40 @@ in stdenv.mkDerivation rec {
     imagemagick
   ];
 
-  installPhase = ''
-    runHook preInstall
+  installPhase =
+    let
+      systemDir = {
+        x86_64-linux = "System64";
+        i686-linux = "System";
+      }.${stdenv.hostPlatform.system} or (throw "unsupported system: ${stdenv.hostPlatform.system}");
+    in ''
```

Here in the first commit, if I were you I'd unindent this in 1 level, so that the diff will be a bit smaller. i.e:

```suggestion
  installPhase = let
    systemDir = {
      x86_64-linux = "System64";
      i686-linux = "System";
    }.${stdenv.hostPlatform.system} or (throw "unsupported system: ${stdenv.hostPlatform.system}");
  in ''
```